src/components/Toolbar.js

```diff
@@ -1,6 +1,32 @@
+/* eslint-disable fp/no-let */
 import React from 'react'
 import { connect } from 'react-redux'
 import { shortcutById } from '../shortcuts'
+import {
+  overlayReveal,
+  overlayHide,
+  overlayUpdate
+} from '../action-creators/toolbar'
+import { CSSTransition, TransitionGroup } from 'react-transition-group'
+
+const isTouchEnabled = () => {
+  return (
+    'ontouchstart' in window ||
+    navigator.maxTouchPoints > 0 ||
+    navigator.msMaxTouchPoints > 0
+  )
+}
+
+(() => {
+  window.addEventListener('mouseup', () => {
+    clearHoldTimer()
+  })
+  if (isTouchEnabled()) {
```

Comment:
I would assume there is no reason to check for this condition since `touchend` would only be called if the device is touch enabled.

Reply:
I forget to remove some of the leftover conditions sometimes, I'll do better.